pkgs/development/tools/xcode-install/default.nix

```diff
@@ -0,0 +1,15 @@
+{ bundlerApp }:
```

Comment:
```suggestion
{ lib, bundlerApp }:
```

`lib` is not in scope.

Reply:
Ah, you're right, didn't change this after adding the metadata. Whoops!